### PR TITLE
enhacement: copy zkeyfiles to the dist folder

### PR DESCRIPTION
**Pull request by iskdrews**
## Explanation

Copy the zkeyFiles of both the `RLN` and the `Semaphore` protocols to the dist, so demo (DApp) doesn't need to load the zkeyfiles. 

## More Information

- See #119 

## Pre-Merge Checklist

- [x] PR template is filled out
- [ ] **IF** this PR fixes a bug, a test that _would have_ caught the bug has been added
- [x] PR is linked to the appropriate GitHub issue
- [x] PR has been added to the appropriate release Milestone

### + If there are functional changes:

- [ ] Manual testing complete & passed
- [ ] "Extension QA Board" label has been applied

> PR template source from [github.com/MetaMask](https://github.com/MetaMask)

### CODE REVIEW

File: src/background/zk-keeper.ts [hunk 1]
```diff
@@ -125,9 +126,15 @@ export default class ZkKeeperController extends Handler {
       RPCAction.PREPARE_SEMAPHORE_PROOF_REQUEST,
       LockService.ensure,
       this.zkValidator.validateZkInputs,
-      async (payload: SemaphoreProofRequest, meta: any) => {
+      async (payload, meta: any) => {
```
Comment (0xmad) [inline src/background/zk-keeper.ts:129]: I'd keep type here.

**Review by 0xmad — COMMENTED** [review]
Just a minor typescript issue

File: src/background/zk-keeper.ts (hunk 1 above)
Comment (iskdrews) [reply src/background/zk-keeper.ts]: Resolved.

**Comment by 0xmad** [comment]
Seems there are few issues:
When I click `Circuit`, `ZKey` - I can't download the file like it was before.
`Merkle` doesn't work as well but seems it's an old bug.

image

File: zkeyFiles/rln/verification_key.json [hunk 2]
```diff
@@ -0,0 +1,119 @@
+{
```
Comment (0xmad) [inline zkeyFiles/rln/verification_key.json:1]: Let's remove zkeyFiles json files from prettier ignore file and run prettier.

**Review by 0xmad — APPROVED** [review]
Looks good to me, just some minor code style comments :)

**Pull request merged**